draft-rbw-add-encrypted-dns-forwarders.md

```diff
@@ -357,16 +357,18 @@ making it impossible to identify whether the delegated credential is
 issued to the intended CPE or an "evil-twin" CPE. This drawback can be corrected
 by enhancing {{?RFC9345}} to include a string that uniquely identifies the
 delegated credential (e.g., including hash of customer id or other unique identifier
-in the FQDN to result in "HASH.cpe.example.com").
-
+in the FQDN such as "HASH.cpe.example.com").  This document assumes such an
+enhancement to {{?RFC9345}} has been standardized and deployed.
```

Comment:
```suggestion
> enhancement to {{?RFC9345}} has been standardized and deployed.
```

Reply:
we may want to say "enhancement to {{?RFC9345}} will be standardized and deployed."